frontend/src/lib/Apis.ts

```diff
@@ -25,7 +25,7 @@ import { StoragePath } from './WorkflowParser';
 import { buildQuery } from './Utils';
 
 const v1beta1Prefix = 'apis/v1beta1';
-const token = localStorage.getItem('token') || ''
+const token = localStorage.getItem('token')
```

Comment:
do you need this change?

Reply:
yes, _fetch api is using it.

Reply:
I remember, || '' was added to avoid some compilation or warning. so without this change, what issue you were getting?